src/validation/matrix-sort-type.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Matrix Sort Type Validation
+ */
+
+module.exports = (arg, func) => {
+    if (arg != 'row' || arg != 'col') {
+        throw new TypeError(`Invalid argument received: ${JSON.stringify(arg)}\n'${func}()' only accept 'row' or 'col'!\n`);
```

Comment:
Break long lines into shorter one. Follow 80 chars convention

Reply:
Ohk. Keeping that in mind.